cmd/kubeadm/app/images/images_test.go

```diff
@@ -262,6 +270,18 @@ func TestGetDNSImage(t *testing.T) {
 				},
 			},
 		},
+		{
+			expected: "foo.io/coredns/coredns:v1.10.0",
+			cfg: &kubeadmapi.ClusterConfiguration{
+				ImageRepository: "foo.io/coredns",
+				DNS: kubeadmapi.DNS{
+					Type: kubeadmapi.CoreDNS,
+					ImageMeta: kubeadmapi.ImageMeta{
+						ImageTag: "v1.10.0",
```

Comment:
what does this test case add that we are not testing already?

Reply:
Yes, this test case tests the validity of ImageTag.

Reply:
https://github.com/kubernetes/kubernetes/blob/e218e0ede4eef31027adb6c868b3b6099b25f02b/cmd/kubeadm/app/images/images.go#L59-L61

Line 60 is not covered before.

Reply:
you can just add it to the previous test and expect a custom tag?

Reply:
Thanks for your advice. I'll modify it.